includes/classes/admin/class-admin.php

```diff
@@ -42,7 +42,21 @@ public function init() {
 		/**
 		 * Enqueue scripts.
 		 */
-		public function enqueue_scripts() {
+		public function enqueue_scripts( $hook ) {
+			switch ( $hook ) {
+				case 'edit-tags.php':
+				case 'term.php':
```

Comment:
This should be moved to the PRO plugin codebase. The `board` taxonomy does not exist in the Free plugin. While we may have to duplicate the enqueue code, I'd rather have the two plugins decoupled and separate from each other 👍

Reply:
@MattGeri if you want perfect decoupling then we would need not only the enqueue code in PRO but also the assets there - which will violate DRY. The other option is to create a custom action in the free plugin that will load the assets and call that action from PRO whenever required, which will not violate DRY. This action may come in handy for later changes as well. Let me know what you prefer.

Reply:
Yea, unfortunately that is the case. But decoupling is more important than DRY in this case.

>The other option is to create a custom action in the free plugin that will load the assets and call that action from PRO whenever required

I like this! Good approach 👍